Feature: Adding border option to responsive image template

shredtechular opened the pull request:
This PR fixes #47 .

This update adds in a new ability to pass in a border parameter to the image template in order to draw a 1px solid border around an image.

For now, I coded this feature so that a writer can simple add: `border="true"` to the image include and the template will add the proper border class to the image.  In the future, if additional styles of borders are desired, then we may want to add a little more logic to also give the user the ability to pass in a specific border class.  In other words, please see the table below:

| Parameter Value        | Result           | Complete in this PR?  |
| ------------- | ------------- |:-----:|
| paramter null or not defined at all      | no border around image | Yes |
| false      | no border around image | Yes |
| true      | adds default border class for 1px solid gray border | Yes |
| specific border class (string) | adds specified border class | No, can include now or for future release if desired.  The CSS would have to exist as well for any other use-cases/styles for it to really work |

**Please NOTE:** I included 1 example of the image include with the border turned on for 1 image in the `_posts/2018-04-12-load-test-3.md` article.  I'm not sure if a border is desired on this image or not, but thought it would be good to include an example?  If you'd like me to add that parameter to ALL graph images or remove the 1 example, I can do either of those, but wanted to specifically call that example out.

shredtechular commented:
Seeing a merge conflict on this PR now with some other previous PRs being merged.  Let me resolve and fix this on my end.

mtlynch commented:
Cool, this looks good. Can you add `border="true"` to all the existing images where it makes sense? I trust your judgment on which images need it.

shredtechular commented:
Absolutely, I can definitely add that parameter where it makes sense on existing images.  I will do that along with fixing the conflict.

shredtechular commented:
OK, I have rebased this branch now to remove the conflicts and also added `border="true"` to existing images where it makes sense in c874c43.

This PR is ready for another round of review when ready.
